mrbgems/mruby-sprintf/src/sprintf.c

```diff
@@ -999,7 +999,8 @@ mrb_str_format(mrb_state *mrb, mrb_int argc, const mrb_value *argv, mrb_value fm
       case 'A': {
         mrb_value val = GETARG();
         double fval;
-        int i, need = 6;
+        int i;
```

Comment:
I'd rather replace all `int` to `mrb_int` in the function.

Reply:
Fixed. But because the address of `i` was used in `frexp` call, a new temporary variable has been introduced.